app/assets/stylesheets/_badges.scss

```diff
@@ -26,4 +26,23 @@
     text-decoration: none;
     background-color: darken($label-security_alert-color, 10% );
   }
-}
\ No newline at end of file
+}
+
+.pending {
```

Comment:
can we keep fills in the `_fills.scss` file?

Reply:
made the changes.